### PR TITLE
Add scripts for Windows LUCI recipe

### PR DESCRIPTION
**Pull request by stuartmorgan**
See https://github.com/flutter/flutter/issues/89373 for context. This
sets up the script and config file to replicate the current plugins.py
recipe's behavior using the planned generic recipe.
    
These are currently unused, but are being landed first so that the
generic recipe has something to test against.
    
Part of https://github.com/flutter/flutter/issues/89373

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Note that unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I [updated pubspec.yaml](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates) with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated CHANGELOG.md to add a description of the change.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code

### CODE REVIEW

File: .ci/scripts/drive_examples_win32.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (CaseyHillers) [inline .ci/scripts/drive_examples_win32.sh:1]: TIL our windows bots have bash, sounds like I can do some cleanup :-)

**Review by CaseyHillers — APPROVED** [review]
LGTM!

@christopherfujino has put a lot of thought into the single recipe design, so I'll let him check this aligns with it

File: .ci/scripts/drive_examples_win32.sh (hunk 1 above)
Comment (stuartmorgan) [reply .ci/scripts/drive_examples_win32.sh:1]: This wouldn't actually work on the Windows bot presumably (they run it with `bash foo.sh`), but I kept it for consistency with the existing bash scripts in the repo.

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by stuartmorgan** [comment]
`submit-queue` is wrong again; landing manually.

**Pull request merged**